reviews/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from reviews.models        import Review
+from django.http           import JsonResponse, request
+from django.views          import View
+from users.models          import User
+from products.models       import Product
+from users.utils           import login_decorator
+
+class ReviewView(View):
+    @login_decorator    
+    def post(self,request):
+        try:
+            data = json.loads(request.body)
+
+            if data["title"] == "" :
+                return JsonResponse({"MESSAGE":"NULL_REVIEWS"}, status = 400)
+
+            if data["size_rating"] == "" or data["color_rating"] == "" or data["delivery_rating"] == "" :
+                return JsonResponse({"MESSAGE":"NULL_RATING"}, status = 400)
+
+            rating_max = 6
+            if data["size_rating"] >= rating_max or data["color_rating"] >= rating_max or data["delivery_rating"] >= rating_max :
+                return JsonResponse({"MESSAGE":"RATING_INPUT_ERROR"}, status = 400)
+
+            Product.objects.filter(id=request.product.id).exists()
+
+            Review.objects.create( 
+                user            = request.user,
+                product         = data["product_id"],
+                size_rating     = data["size_rating"],
+                color_rating    = data["color_rating"],
+                delivery_rating = data["delivery_rating"],
+                title           = data["title"],
+                comment         = data["comment"]
+            )
```

Comment:
```suggestion

            if not Product.objects.filter(id=request.product.id).exists():
                return 

            Review.objects.create( 
                user            = request.user,
                product_id         = data["product_id"],
                size_rating     = data["size_rating"],
                color_rating    = data["color_rating"],
                delivery_rating = data["delivery_rating"],
                title           = data["title"],
                comment         = data["comment"]
            )
```
1. 존재 유무를 판단하는 로직이 중간에 조금 빠진것 같네요. 확인해 주세요.
2. Product 객체를 넣어주는 것이 아닌 product의 id를 직접 넣기 때문에 product_id라는 속성을 사용해야 합니다.